scipy/stats/tests/test_distributions.py

```diff
@@ -211,7 +211,8 @@ def test_vonmises_expect():
 def test_vonmises_fit_all(kappa, loc):
```

Comment:
It would be good to add a parameterized test that uses `_assert_less_or_close_loglike` to compare against the generic fit. The current tests just demonstrate that the fit is reasonable, but not that is a maximum likelihood estimate or at least better than the generic `fit` behavior (with `fscale=1`, of course).

Reply:
Can do that in a follow-up.